_posts/2019-07-02-very-large-svds.md

```diff
@@ -0,0 +1,292 @@
+---
+layout: post
+title: Very Large SVDs
+tagline: Dask + CuPy + Zarr + Genomics
+author: Alistair Miles (Oxford Big Data Institute), Ben Zaitlen (NVIDIA), Matthew Rocklin (Coiled)
```

Comment:
```suggestion
author: Ben Zaitlen (NVIDIA), Matthew Rocklin (Coiled), Alistair Miles (Oxford University)
```

I don't deserve to be first author here :)

Reply:
I think we should do it alphabetically by first name.  I believe that happens to coincide with the current ordering of names

Reply:
That would feel a bit strange for me, my contribution to this nice work is very minor.

Reply:
changed the author order